### PR TITLE
Fixed the TOC of the main Forms article

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: forms.rst [hunk 1]
```diff
@@ -696,10 +696,9 @@ Learn more
     :glob:
 
     form/*
-
-* :doc:`/controller/upload_file`
-* :doc:`/reference/forms/types`
-* :doc:`/http_cache/form_csrf_caching`
+    /controller/upload_file
```
Comment (kunicmarko20) [inline forms.rst]: Is that slash at the beginning needed? since `form/*` doesn't have it?

Comment (javiereguiluz) [reply forms.rst]: Good catch! I've added the leading slash to forms.

Also, it looks like all formats are supported. Most `toctree` add a leading `/`, some don't. Most `toctree` don't add `.rst` suffix. Some do.

**Review by kunicmarko20 — APPROVED** [review]

**Comment by kunicmarko20** [comment]
Looks like there is an issue with `plaformsh`

**Comment by javiereguiluz** [comment]
@kunicmarko20 yes, all builds are failing at the moment. We'll contact them. Cheers.

**Pull request closed without merging**